recommended.js

```diff
@@ -107,6 +107,12 @@ module.exports = {
         '@lwc/lwc/no-document-query': 'error',
         '@lwc/lwc/no-inner-html': 'error',
         '@lwc/lwc/no-leading-uppercase-api-name': 'error',
+        '@lwc/lwc/valid-api': [
```

Comment:
What do you think about also enabling: https://github.com/salesforce/eslint-plugin-lwc/blob/v0.11.0/docs/rules/prefer-custom-event.md ?

Reply:
agreed! is done in 5d8d309